community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacadeFactory.java

```diff
@@ -88,6 +88,7 @@
         // Kept here to have it not be publicly documented.
         public static final Setting<String> lock_manager = setting( "unsupported.dbms.lock_manager", Settings.STRING, "" );
         public static final Setting<String> record_format = setting( "unsupported.dbms.record_format", Settings.STRING, "" );
+        public static final Setting<String> auth_manager = setting( "unsupported.dbms.auth_manager", Settings.STRING, "" );
```

Comment:
This should be in `GraphDatabaseSettings` and be annotated as `@Internal`, please.

Our other security-related settings live under `dbms.security.*` (or `unsupported.dbms.security.*`) so this should probably be there too.